draft-ietf-tls-tls13.md

```diff
@@ -1857,24 +1832,21 @@ in the first flight, the rest of the handshake uses the same messages.
                                                          ServerHello
                                                         + early_data
                                                          + key_share
-                                               {EncryptedExtensions}
-                                               {CertificateRequest*}
-                                                          {Finished}
+                                               [EncryptedExtensions]
+                                               [CertificateRequest*]
+                                                          [Finished]
                                    <--------     [Application Data*]
-         {Certificate*}
-         {CertificateVerify*}
-         {Finished}                -------->
+         [Certificate*]
+         [CertificateVerify*]
+         [Finished]                -------->
 
          [Application Data]        <------->      [Application Data]
 
                *  Indicates optional or situation-dependent
                   messages that are not always sent.
 
                () Indicates messages protected using keys
-                  derived from the static secret.
-
-               {} Indicates messages protected using keys
-                  derived from the ephemeral secret.
+                  derived from the PSK.
 
                [] Indicates messages protected using keys
                   derived from the master secret.
```

Comment:
Rather than split keys based on what they are derived from, use early_traffic_secret, handshake_traffic_secret, and traffic_secret_0 ?